django_sites_extensions/models.py

```diff
@@ -1,8 +1,26 @@
 """ Django Sites framework models overrides """
-from django.contrib.sites.models import Site, SiteManager
+import datetime
+
+from django.contrib.sites.models import Site, SiteManager, SITE_CACHE
 from django.core.exceptions import ImproperlyConfigured
 
 
+# Dict which maps Site id/domain to a datetime
+# when the corresponding when the corresponding
+# Site model cached in SITE_CACHE should be flushed.
+SITE_CACHE_TIMEOUTS = {}
+
+
+def get_site_cache_ttl():
+    """
+    Get the SITE_CACHE_TTL timedelta.
+
+    Defaults to 5 minutes if SITE_CACHE_TTL has not been configured in application settings.
+    """
+    from django.conf import settings
```

Comment:
Is it necessary (or better) to do this local import vs. moving to the top of the module?  I know we try to avoid these unless absolutely required due to circular dependencies or performance.